sources/packages/backend/apps/db-migrations/src/sql/SFASPartTimeApplications/Add-col-py-maximum-support.sql

```diff
@@ -0,0 +1,39 @@
+-- Add column cspt_award for SFAS Part Time Application.
+ALTER TABLE 
+  sims.sfas_part_time_applications 
+ADD 
+  COLUMN IF NOT EXISTS cspt_award NUMERIC(10);
+
+COMMENT ON COLUMN sims.sfas_part_time_applications.cspt_award IS 'CSPT Award Amount (sail_extract_data.sail_cspt_amt).';
+
+-- Add column csgd_award for SFAS Part Time Application.
+ALTER TABLE 
+  sims.sfas_part_time_applications 
+ADD 
+  COLUMN IF NOT EXISTS csgd_award NUMERIC(10);
+
+COMMENT ON COLUMN sims.sfas_part_time_applications.csgd_award IS 'CSGD Award Amount (sail_extract_data.sail_ptdep_amt).';
```

Comment:
`sail_extract_data.sail_csgd_amt` ?

Reply:
I saw this too, but I'm using `sail_extract_data.sail_ptdep_amt` because it is on the document.
![image](https://github.com/bcgov/SIMS/assets/148148914/af3e63fc-36aa-42ea-beca-f3e4d8c9fbd5)

Reply:
As @lewischen-aot shared, the comment should follow the documentation from the bridge.